app/code/Magento/InventoryConfigurationApi/Api/Data/StockItemConfigurationInterface.php

```diff
@@ -64,11 +66,6 @@ interface StockItemConfigurationInterface extends \Magento\Framework\Api\Extensi
      */
     public function isQtyDecimal(): bool;
 
-    /**
```

Comment:
Removing methods from the interface would break BC and therefore postpone this feature to the later releases. Should we look for ways to preserve methods?

Reply:
I removed those methods to follow the [directive here that asks for immutability](https://github.com/magento-engcom/msi/wiki/Stock-and-Source-Configuration-design#programming-interface-query-api). Let me know which way you prefer to follow.

Reply:
General approach for immutability is still in works, so I'm not sure if we should remove methods right away. Still, it's possible that once this PR is ready we would be working on a release which would allow some changes to the interfaces.

In short, I'd keep it at this point. We can always remove it down the road if it looks like a good idea then